pfcpiface/pfcpiface.go

```diff
@@ -20,6 +20,15 @@ const (
 	MaxItems = 10
 )
 
+type sessRecord struct {
+	pdrs     [MaxItems]pdr
+	fars     [MaxItems]far
+	pdrCount uint8
+	farCount uint8
```

Comment:
Can this be slice and count is already available? Any specific reason for array?

Reply:
Converted to slice.